tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/BallerinaLexer.g4

```diff
@@ -345,26 +345,6 @@ QuotedStringLiteral
     :   '"' StringCharacters? '"'
     ;
 
-SymbolicStringLiteral
```

Comment:
@NatashaWso2 Shall we revert as we discussed in other PR. 

@NipunaRanasinghe Please note.

Reply:
@hasithaa reverted the changes with a574eb2